moped-editor/src/components/GridTable/Filters.js

```diff
@@ -334,13 +275,56 @@ const Filters = ({
     setIsOrToggleValue(isOr);
   };
 
+  // Shared style classes
+  const formControlStyle = {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  };
```

Comment:
@tillyw i looked at this again, and I think that these can be moved back outside of the component definition so they aren't recreated on each render. Either by using the callbacks on the object level `const style = (theme) => {...needsTheme}`, the attribute level `const style = {...otherStyles, (theme) => {...needsTheme}}, or shorthand `const style = { margin: 1 }`.

I know there are a million and one ways to implement this 😵‍💫, but these ones would work around the need for these objects to live inside the component.

Reply:
ok done! i'll default to doing it this way (at the object level for consistency) from now on so we can use imports only when necessary